rules/enforce-else-after-else-if.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview Encourage else blocks after else-if blocks
+ * @author Michelle Pokrass <michellepokrass@gmail.com>
+ */
+
+'use strict';
+
+module.exports = {
+
+    meta: {
+        docs: {
+            description: "Encourage else blocks after else-if blocks",
+            recommended: true,
+            type: "error"
+        },
+
+        schema: [],
+        fixable: "code"
+    },
+
+    create(context) {
+        function inspectElseIfStatement(node) {
+            // if the else if does not have an alternate, that means there is
+            // no else - report an error
+            if (!node['alternate']) {
+                context.report({
+                    node: node,
+                    message: 'else if statement not followed by else'
```

Comment:
Please change message to `"else if" statement must be followed by an "else" statement`